system_strings/sysmes_text.en

```diff
@@ -1390,7 +1390,7 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
 判断が甘かった。^巨人の腕は傷付きさえしない。^この戦力差をどうする……？
 光体には傷一つ付かなかったが、^今を逃せばチャンスはない。^果敢にも巨人に挑む。
 失敗した以上、猶予はない。^腕の切断を諦めてでも、^先輩を解放しなくては……！
-光体に吹き飛ばされ、地上へ。^落下した先に待っていたものは、^圧倒的な自然の驚異だった。
+I'm blown to the ground by the^Luminary. What awaits me on the^surface is an overwhelming^miracle of nature.
```

Comment:
Miracle isn't quite right. It's basically a surprise. Not sure how to edit rn

Reply:
> What awaits me on the surface is an shocking profusion of nature.

Reply:
a*, but that works